manifests/overlays/prod/applications/thoth-station/ocp4-stage/stage-thoth-graph-schema-update.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: ocp4-stage-thoth-graph-schema-update
+spec:
+  project: thoth-stage
+  source:
+    repoURL: 'https://github.com/thoth-station/thoth-application.git'
+    path: graph-schema-update/overlays/ocp4-stage
+    targetRevision: master
+  destination:
+    server: 'https://api.ocp4.prod.psi.redhat.com:6443'
+    namespace: thoth-infra-stage
+  syncPolicy:
+    automated:
+      selfHeal: true
+      prune: true
```

Comment:
You may need to disable validation as well. ArgoCD usually fails on some nitpicks with OpenShift objects

Reply:
@anishasthana was the comment related to the `validate` option under `syncOptions`? I was under the impression that it is disabled my default.

Reply:
I was of the impression that it is enabled by default. @HumairAK thoughts?

Reply:
Hrmm honestly I'm not sure, can't find anything in the docs. I would just add it to eliminate doubt.

Reply:
ok , thanks both of you. update it as well.